src/EditorFeatures/CSharpTest/UseExpressionBody/Refactoring/UseExpressionBodyForAccessorsRefactoringTests.cs

```diff
@@ -74,6 +95,46 @@ int Goo
 }", parameters: new TestParameters(options: UseExpressionBodyForAccessors_ExpressionBodyForProperties));
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseExpressionBody)]
+        public async Task TestOfferedIfUserPrefersExpressionBodiesWithoutDiagnosticAndInBlockBody()
+        {
+            await TestInRegularAndScript1Async(
+@"class C
+{
+    int Goo
+    {
+        get
+        {
+            return [||]Bar();
+        }
+    }
+}",
+@"class C
+{
+    int Goo => Bar();
```

Comment:
💭 It's not obvious to me if this is right or wrong, but it also doesn't seem like a common configuration case.